kolibri/content/test/test_annotation.py

```diff
@@ -0,0 +1,209 @@
+import tempfile
+
+from django.core.management import call_command
+from django.test import TransactionTestCase
+
+from kolibri.content.models import ContentNode, File, LocalFile
+from kolibri.content.utils.annotation import (
+    set_local_file_availability, set_local_file_availability_from_disk,
+    set_leaf_node_availability_from_local_file_availability, recurse_availability_up_tree
+)
+
+from le_utils.constants import content_kinds
+
+from mock import patch
+
+from .sqlalchemytesting import django_connection_engine
+
+def get_engine(connection_string):
+    return django_connection_engine()
+
+class AnnotationFromLocalFileAvailability(TransactionTestCase):
+
+    fixtures = ['content_test.json']
+
+    def test_all_local_files_available(self):
+        LocalFile.objects.all().update(available=True)
+        with patch('kolibri.content.utils.sqlalchemybridge.get_engine', new=get_engine):
+            set_leaf_node_availability_from_local_file_availability()
```

Comment:
Yeah, for this test that can definitely be the case. In some of the other tests, I think it got a bit more complicated.

Also, when you patch a class, the mocks don't get passed into setup, which is a pain.